lib/main/traceLogger.ts

```diff
@@ -0,0 +1,258 @@
+import { trace, SpanStatusCode, SpanKind } from '@opentelemetry/api'
+import { Resource } from '@opentelemetry/resources'
+import { SemanticResourceAttributes } from '@opentelemetry/semantic-conventions'
+import { NodeSDK } from '@opentelemetry/sdk-node'
+import { BatchSpanProcessor } from '@opentelemetry/sdk-trace-base'
+
+// Simple console exporter
+class ConsoleSpanExporter {
+  export(spans: any[], resultCallback: (result: any) => void) {
+    spans.forEach(span => {
+      console.log('[OpenTelemetry]', {
+        name: span.name,
+        traceId: span.spanContext().traceId,
+        spanId: span.spanContext().spanId,
+        startTime: span.startTime,
+        endTime: span.endTime,
+        attributes: span.attributes,
+        events: span.events,
+        status: span.status,
+      })
+    })
+    resultCallback({ code: 0 })
+  }
+
+  shutdown(): Promise<void> {
+    // No cleanup needed for console exporter
+    return Promise.resolve()
+  }
+}
+
+// Initialize OpenTelemetry SDK
+const sdk = new NodeSDK({
+  resource: new Resource({
+    [SemanticResourceAttributes.SERVICE_NAME]: 'ito-app',
+    [SemanticResourceAttributes.SERVICE_VERSION]: '1.0.0',
```

Comment:
Doesn't have to be done in this ticket, but we should absolutely get inject this based on the actual release version. Would you file a follow up ticket for that? We probably just need a simple text file or .env variable we can update as part of the release workflow

Reply:
Yeah, once the versioning system is more solid, we should definitely propagate the version here.